### ISSUE TITLE
Can this please be updated to version 11?

### ISSUE THREAD

**Issue by Orphansmith** [free-beer/black-sword-hack#15]
It looks like this hasn't been updated for a while.

**Comment by Orphansmith** [comment]
If I donate on the kofi page, can this be updated?

**Comment by free-beer** [comment]
Sorry, what exactly are you seeing in terms of issues in relation to version 11?

**Comment by Orphansmith** [comment]
I'm unable to run the system in Foundry version 11 on Forge-VTT.

**Comment by free-beer** [comment]
Okay, I'll see if I can take a look. In the meantime, Forge lets you set the Foundry version in the world configuration. Rollback to version 10 there if you need to use it.

**Comment by free-beer** [comment]
So I took a look on the Forge. I've made a release of the package that brought the version number up to v1.1.3 but the Forge is only listing version v1.0.0. That might take a little time to update as it's only just released but I'm guessing, if they are that far behind the latest version, that it's not going to update. That being the case, you'll have to contact the Forge to see what the issue is with why they are only listing an older version.

I have Foundry v11.315 installed locally and I've not found any issues with the system after giving it a very quick shakedown. If you get it working and spot something then let me know.

**Comment by Orphansmith** [comment]
Great, thank you!

**Comment by free-beer** [comment]
Been playing around and it seems the system version might be tied to the default version of Foundry that you're using on the Forge. I went to the Game Manager interface (https://eu.forge-vtt.com/setup#manager=1 for me). Here I checked that the default Foundry version for my games was set to the latest (v11, Patch 13, Build 315). Then I went to Bazaar -> Systems and selected the Installed quick filter on the left hand column to just list my installed systems. There was an update option then on the systems that, when I clicked on it for the Black Sword Hack, updated the package version to 1.1.3. When accessing the game I then got a confirmation about the upgrade. Once upgraded it appeared to work fine for me.
